Dockerfile-multiarch

```diff
@@ -0,0 +1,19 @@
+# Dockerfile for ghostunnel/ghostunnel, used for multi-arch release builds.
+#
+# To run ghostunnel from the image (for example):
+#     docker run --rm ghostunnel/ghostunnel --version
+
+# Work out which binary in dist/ to use
+FROM debian:buster-slim as build
+ARG TARGETPLATFORM
+ARG VERSION
+ADD dist/ dist/
+RUN ARCH=$(echo ${TARGETPLATFORM} | cut -d'/' -f2) && \
+    cp dist/ghostunnel-${VERSION}-linux-${ARCH}-with-pkcs11 /usr/bin/ghostunnel
+
+# Create the actual image using a multi-stage build
+FROM debian:buster-slim
```

Comment:
Let's try to keep this as close as possible to the current image, in case anyone depends on what's in here. Can we switch this to Alpine and make sure libtool/curl is installed?

Reply:
Sure we can do. I changed to Debian so we could use the existing binaries to avoid compiling everything twice. However I don't mind going back to compiling in the container build if that's what you prefer. That would be simpler...